.buildkite/pipeline.yml

```diff
@@ -1,13 +0,0 @@
-steps:
```

Comment:
Buildkite used to be faster than GitHub actions. I guess we can trade CI checks speed since we're going to build the binaries in GCB. @ecp88 what do you think?

Reply:
Decision: we will remove buildkite once we completely shut down all the `nearcore`-based Indexer for Explorer nodes. Currently we need to update the Buildkite to work on `master-nearcore` branch to build binaries
cc @ecp88